src/components/editor/TextEditor/AddImage/ImageItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactComponent as Trash } from 'assets/svg/trash.svg';
+import { useRef, useState } from 'react';
+import useOnClickOutside from 'utils/hooks/useOnClickOutside';
+import styles from '../TextEditor.module.scss';
+
+interface Props {
+  value: string,
+  removeImage: (value: string) => void,
+}
+
+export default function ImageItem({ value, removeImage }: Props) {
+  const [buttonOpened, setButtonOpened] = useState(false);
```

Comment:
여기 레포에 `useBooleanState` 가 있지 않나욤!?
고걸 사용하면 좀더 가독성이 높아질것 같아요

```suggestion
  const [opened, open, close] = useBooleanState(false);
  const imageRef = useRef<HTMLInputElement>(null);
  
  useOnClickOutside(imageRef, close);

  return (
    <>
      {opened && (
        <button
          type="button"
          aria-label="trash"
          className={styles.imageContainer__button}
          onClick={() => removeImage(value)}
        >
          <Trash />
        </button>
      )}
      <input
        type="image"
        className={styles.imageContainer__image}
        src={value}
        alt="이미지"
        ref={imageRef}
        onClick={open}
      />
    </>
```

Reply:
적용했습니다!